### DISCUSSION TITLE
How can I calculate the position of the atom?

### DISCUSSION THREAD

**Discussion by Hegel991** [dschick/udkm1Dsim#142] [Q&A]
When I use the method "add_atom(atom, position)" to build a unit cell, I am quite confused that how to calculate or confirm the position of the atom. Are there any tips or instructions for this? Thanks!!!

**Comment by dschick** [comment]
Hi @Hegel991,

thanks for asking. There is an [example in the docs](https://udkm1dsim.readthedocs.io/en/latest/examples/structure.html#Unit-Cells) which should include most of the details.
It is important to note, that the position is relative to the actual c-axis of the unit cell.
Since the simulations are 1D only, the position in the a-b-plane is irrelevant.

Hope that helps. Let me know if you have any further questions.

Best

Daniel

**Comment by Hegel991** [comment]
Hi @dschick,

Thanks for answering. It helps. In addition, this is a great job! 🤩 

Best 

Hegel991

**Comment by dschick** [comment]
Great & thanks!

You are very welcome
